custom-words.txt

```diff
@@ -2139,3 +2139,8 @@ ziplist
 Zoho
 zoneinfo
 zset
+testfailover
+Orignal
+Onprem
+targetv
+Seleted
```

Comment:
This is a typo, not something to ignore

Reply:
Opened https://github.com/Azure/azure-rest-api-specs/pull/14120 to suppress for the single version and flag again on the next